dotcom-rendering/src/components/SubheadingBlockComponent.stories.tsx

```diff
@@ -0,0 +1,131 @@
+import { css } from '@emotion/react';
+import { ArticleDesign, ArticleDisplay, Pillar } from '@guardian/libs';
+import { headline } from '@guardian/source-foundations';
+import type { Decorator, StoryObj } from '@storybook/react';
+import { splitTheme } from '../../.storybook/decorators/splitThemeDecorator';
+import { SubheadingBlockComponent } from './SubheadingBlockComponent';
+
+const standardFormat = {
+	design: ArticleDesign.Standard,
+	display: ArticleDisplay.Standard,
+	theme: Pillar.News,
+};
+
+const immersiveFormat = {
+	...standardFormat,
+	display: ArticleDisplay.Immersive,
+};
+
+/** Mocking the styles normally inherited via ArticleBody component */
+const GlobalStylesDecorator =
+	(format: ArticleFormat): Decorator =>
+	(Story) => (
+		<div
+			css={css`
+				h2:not([data-ignore='global-h2-styling']) {
+					${format.display === ArticleDisplay.Immersive
+						? headline.medium({ fontWeight: 'light' })
+						: headline.xxsmall({ fontWeight: 'bold' })};
+				}
+			`}
+		>
+			{Story()}
+		</div>
+	);
+
+const StoryWrapper = ({ children }: { children: React.ReactNode }) => (
+	<div
+		css={css`
+			hr {
+				border: 0.5px dashed;
+			}
+
+			margin: 10px;
+
+			* {
+				margin-bottom: 10px;
+			}
+		`}
+	>
+		{children}
+	</div>
+);
+
+const meta = {
+	component: SubheadingBlockComponent,
+	title: 'Components/SubheadingBlockComponent',
+	render: (args: React.ComponentProps<typeof SubheadingBlockComponent>) => {
+		return (
+			<StoryWrapper>
+				<SubheadingBlockComponent
+					format={args.format}
+					html="<h2>Basic subheading</h2>"
+				/>
+				<SubheadingBlockComponent
+					format={args.format}
+					html="<h2>Subheading <a href='/'>with anchor</a></h2>"
+				/>
+				<SubheadingBlockComponent
+					format={args.format}
+					html="<h2>Subheading with HTML comment<!-- HTML comment--></h2>"
+				/>
+				<SubheadingBlockComponent
+					format={args.format}
+					html="Subheading text only (no HTML)"
+				/>
+
+				<hr />
+			</StoryWrapper>
+		);
+	},
+	args: {
+		html: '<h2>Subheading</h2>',
+		format: standardFormat,
+	},
+};
```

Comment:
Should this be?

```suggestion
const meta = {
	component: SubheadingBlockComponent,
	title: 'Components/SubheadingBlockComponent',
	render: (args) => {
		return (
			<StoryWrapper>
				<SubheadingBlockComponent
					format={args.format}
					html="<h2>Basic subheading</h2>"
				/>
				<SubheadingBlockComponent
					format={args.format}
					html="<h2>Subheading <a href='/'>with anchor</a></h2>"
				/>
				<SubheadingBlockComponent
					format={args.format}
					html="<h2>Subheading with HTML comment<!-- HTML comment--></h2>"
				/>
				<SubheadingBlockComponent
					format={args.format}
					html="Subheading text only (no HTML)"
				/>

				<hr />
			</StoryWrapper>
		);
	},
	args: {
		html: '<h2>Subheading</h2>',
		format: standardFormat,
	},
} satisfies Meta<typeof SubheadingBlockComponent>;
```

So that the `render` and `args` fields pick up the correct types?